packages/runtime/src/modules/RequestModule.ts

```diff
@@ -114,6 +114,29 @@ export class RequestModule extends RuntimeModule {
             return;
         }
 
+        const otherRequestsThatWouldLeadToARelationship = (
+            await services.consumptionServices.incomingRequests.getRequests({
+                query: {
+                    "source.type": "RelationshipTemplate",
+                    status: [LocalRequestStatus.Open, LocalRequestStatus.DecisionRequired, LocalRequestStatus.ManualDecisionRequired],
+                    peer: template.createdBy
+                }
+            })
+        ).value;
+        if (otherRequestsThatWouldLeadToARelationship.length !== 0) {
+            this.logger.info(
+                `There is already an open Request for a RelationshipTemplate that would lead to a Relationship with the creator of the RelationshipTemplate '${template.id}'. Skipping creation of a new Request.`
```

Comment:
I'd assume that people know that open requests lead to relationships and simplify to `There is already an open Request with the creator of the RelationshipTemplate '${template.id}'. Skipping creation of a new Request.` Also the `Skipping creation of a new Request.` can be added above in the "existing requests from same template" case

Reply:
Actually the first part can not be simplified as there are also requests where answering would turn into a mesage. Second is good.

Reply:
If answering the request were to turn into a message, there would be a relationship which would be caught beforehand

Reply:
But this is not visible in this Info message. You have to Look in the Code to know that.

Reply:
Alright, it's fine, but how do I put it - seems consistent with the enmeshed documentation style of immediately giving a lot of information rather than giving just enough to proceed

Reply:
What the hell am I reading here.